bigframes/remote_function.py

```diff
@@ -202,10 +202,22 @@ def create_bq_remote_function(
             OPTIONS (
               endpoint = "{endpoint}"
             )"""
+
         logger.info(f"Creating BQ remote function: {create_function_ddl}")
+
+        # Make sure the dataset exists
```

Comment:
Any tests needed here?

Reply:
There are large system tests where we provide a different dataset explicitly than the one that was created in the session. https://github.com/googleapis/python-bigquery-dataframes/blob/main/tests/system/large/test_remote_function.py